tfjs-converter/src/version_test.ts

```diff
@@ -14,17 +14,17 @@ describe('tfjs-core version consistency', () => {
   it('dev-peer match', () => {
     const tfjsCoreDevDepVersion =
         // tslint:disable-next-line:no-require-imports
-        require('../package.json').devDependencies['@tensorflow/tfjs-core'];
+        require('tfjs-converter/package.json').devDependencies['@tensorflow/tfjs-core'];
```

Comment:
Same here as in `metadata_test.ts`. Using any generated files breaks this import for some reason.